docs/README.md

```diff
@@ -1,3 +1,7 @@
+<style>
+#main_content {max-width: 700px;}
```

Comment:
I was able to reproduce. On a Mac with Chrome. What you really want is to override `max-width` of `.inner` to, say, 800px because currently it's 640px. Not great, but...

Reply:
Done.